DatadogRUM/Sources/Instrumentation/RUMCommandSubscriber.swift

```diff
@@ -35,7 +35,7 @@ internal final class GlobalRUMCommandSubscriber: RUMCommandSubscriber {
     ///
     /// - Parameter command: The RUM command to process.
     func process(command: RUMCommand) {
-        let subscriber = Global.rum as? RUMCommandSubscriber
+        let subscriber = RUMMonitor.shared() as? RUMCommandSubscriber
```

Comment:
Might be a personal preference, but I have the same comment about using singleton reference directly.

One argument to back this is that we'll have easier way to test this if we have some custom monitor's behaviour impacting the subscriber.

Reply:
Actually, I can now remove the `GlobalRUMCommandSubscriber` completely 🎉